spec/values/github_session_spec.rb

```diff
@@ -81,4 +81,50 @@ def create_org_response(id, org, avatar_url, role)
       expect(cookies['client_type']).to eq("")
     end
   end
+
+  describe 'fetch_teams_for_user' do
+    context 'when no organizations found' do
+      before { allow(client).to receive(:organizations).and_return([]) }
+
+      it 'returns an empty list' do
+        expect(subject.fetch_teams_for_user('login')).to be_empty
+      end
+    end
+
+    context 'when organizations have teams' do
+      let(:teams) do
+        [
+          team_double('team1'),
+          team_double('team2'),
+          team_double('team3'),
+          team_double('team4')
+        ]
+      end
+
+      let(:organizations) { [double(login: 'org1'), double(login: 'org2')] }
+      let(:org_0_teams) { [teams[0], teams[1]] }
+      let(:org_1_teams) { [teams[2], teams[3]] }
+
+      before do
+        allow(client).to receive(:organizations).and_return(organizations)
+        allow(client).to \
+          receive(:organization_teams)
+          .with(organizations[0].login)
+          .and_return(org_0_teams)
+        allow(client).to \
+          receive(:organization_teams)
+          .with(organizations[1].login)
+          .and_return(org_1_teams)
+      end
+      
+      it 'returns teams' do
+        expect(subject.fetch_teams_for_user('login')).to eq([
+          { name: 'team1' },
+          { name: 'team2' },
+          { name: 'team3' },
+          { name: 'team4' }
+        ])
```

Comment:
Indent the right bracket the same as the first position after the preceding left parenthesis.